Plot language (#2239)
bootstrapworld/curriculum: lessons/Algebra2/exploring-periodic-models/langs/en-us/index.adoc

}

=== Synthesize

> This worksheet had you create a *new type of data visualization*, called a @vocab{line graph}.

Reviewer: ```suggestion
You just created a *new type of data visualization*, called a @vocab{line graph}.
```
